frameworks/helloworld/tests/test_region_awareness.py

```diff
@@ -89,8 +100,6 @@ def test_region_config_update_does_not_succeed():
         [l['value'] for l in info['labels']['labels'] if l['key'] == 'offer_region'][0] == 'USA'
     )
 
-    sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
-
 
 def change_region_config(region_name):
     service_config = sdk_marathon.get_config(config.SERVICE_NAME)
```

Comment:
Should we not wait for a triggered and completed deployment in this case?

Reply:
I don't think so, why?

Reply:
Sorry, I might have been referring to the wrong line here.